Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -477,7 +477,13 @@ void HexEditor::keydown_event(GUI::KeyEvent& event)
         move_and_update_cursor_by(-m_position);
         return;
     }
-    
+
+    if (event.key() == KeyCode::Key_End) {
```

Comment:
Same comment as for previous commit, and I guess that you meant to write "the user" instead of "uer"